CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ None at this time.
 
 ## Notable non-backward compatible changes
 
+- Vi raw mode is now the default and cannot be disabled (issue #864).
```

Comment:
It does not look like a backward incompatible change to me. I see this entry in `RELEASE` file:

```
01-11-26 +The default has compilation mode has been changes so that
              viraw mode will always be on.
```

I can not disable `viraw` mode in version of `ksh` that is shipped with RHEL.

Reply:
Note that "the default compilation mode" simply means that viraw was enabled by default. The second clause of that sentence seems to be independent of the first clause. And looking at the source in the legacy ast.master and ast.beta branches it is not at all obvious you can't disable viraw behavior. Nonetheless, I'm happy to move this statement about `viraw` to the "Other significant changes" section.